### DISCUSSION TITLE
Difficult to navigate to link - have to click on small icon to go the link.

### DISCUSSION THREAD

**Discussion by CarlosOnline** [WebCrateApp/webcrate#51] [General]
Expect that clicking on the tile would navigate me to the link.  Instead I have to find the little icon and click on it.  I don't really need to edit or see the details of the link once I have added it.  I would like to see the primary functionality be navigation instead of details / edit. 

![image](https://user-images.githubusercontent.com/3691895/141519397-4823c12b-f680-4903-85d9-f8540707a238.png)

**Comment by BetaHuhn** [comment]
Hey, thanks for taking the time to open this discussion!

WebCrate has a focus on adding your own context to your links, that's why clicking on the title/link item will open its details. In the future those details may also include more than just the image and your notes and thus I won't change the current behavior. In addition, navigating to the link by clicking the icon is pretty easy already and you get used to it quickly.

We could add the ability to hold a key while clicking anywhere on the link (including the title) to navigate to the URL instead of opening the details. What do you think about that, would that help?

**Comment by CarlosOnline** [comment]
Navigation to Target site is the most important feature to me.  Holding a
key to make that easier is not the best experience.  The bookmarks bar is
still easier to use even if it's not as organized.
